Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -18008,7 +18078,7 @@
             <LayoutButtons Face="AP_GlialReconstitution" Type="Passive" Requirements="AP_HaveHotSRoachSpeed" Row="1" Column="3"/>
             <LayoutButtons Face="AP_OrganicCarapace" Type="Passive" Requirements="AP_HaveHotSRoachHealth" Row="1" Column="4"/>
             <LayoutButtons Face="AP_RapidRegeneration" Type="Passive" Row="2" Column="1"/>
-            <LayoutButtons Face="AP_Ravager" Type="AbilCmd" AbilCmd="AP_MorphToRavager,Execute" Row="2" Column="0"/>
+            <LayoutButtons Face="AP_Ravager" Type="AbilCmd" AbilCmd="AP_MorphToRavager,Execute" Requirements="AP_HaveLair" Row="2" Column="0"/>

```

Comment:
Good change, but did you test that ravagers can still be morphed on Enemy Within and Conviction?

Reply:
I didn't, and you're right, this wouldn't get overwritten on those missions; from what I can tell though, there's no way to add the requirement without modifying the abilities on those missions specifically. I'll just remove it for now and that change can be a different PR.